tjreports/administrator/models/fields/users.php

```diff
@@ -17,7 +21,7 @@
  *
  * @since  1.0.0
  */
-class JFormFieldUsers extends JFormFieldList
+class UserFields extends JFormFieldList
```

Comment:
recheck field class names
i think those still need to be starting with `JFormField`

Reply:
done